composer.json

```diff
@@ -124,6 +125,8 @@
         "codestyle": "phpcs --extensions=php --standard=./ruleset.xml ./",
         "codestyle:ci": "@codestyle --report=checkstyle --report-file=./build/checkstyle-be.xml --no-cache",
         "phpstan": "phpstan analyse -c phpstan.neon --level 6 --memory-limit=512M",
-        "phpstan:ci": "@phpstan --error-format=absolute --no-interaction --no-progress"
+        "phpstan:ci": "@phpstan --error-format=absolute --no-interaction --no-progress",
+        "pest": "./vendor/bin/pest",
```

Comment:
this should replace `test` script command
furthermore, are you sure no bootstrap file is needed ?

Reply:
Removed test command
About bootstrap file you can initialize pest with ./vendor/bin/pest --init but since test files already exist pest can do assertions

Reply:
bootstrap file path is already contained in phpunit.xml, so not needed here